code/game/world.dm

```diff
@@ -188,7 +196,6 @@ GLOBAL_LIST_EMPTY(world_topic_handlers)
 	config = new /datum/configuration()
 	config.load("config/config.txt")
 	config.load("config/game_options.txt","game_options")
-	config.loadsql("config/dbconfig.txt")
 	config.loadoverflowwhitelist("config/ofwhitelist.txt")
```

Comment:
I'm not sure what your plan here is exactly, but `game_options.txt` doesn't exist anymore, so I'd imagine you missed this one.

Reply:
I aint ripped that one apart yet, PR is still WIP